src/pages/ProfilePage/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import smilies from 'assets/smilies.svg';
+import defaultProfile from 'assets/default_profile.svg';
+
+import { getProfile, updatePassword, deleteAccount } from 'redux/actions/userActions';
+import useForm from 'hooks/useForm';
+
+import withMap from 'components/HOCs/withMap';
+import Button from 'components/common/button/Button';
+import FormInput from 'components/common/formInput/FormInput';
+
+const config = {
+  currentPassword: { presence: true, minLength: 6 },
+  password: { presence: true, minLength: 6 },
+  confirmPassword: { presence: true, equalsTo: 'password' }
+};
+
+const ProfilePage = () => {
+  const user = useSelector(({ user: { value } }) => value);
+  const { data, validation, validForm, handleFieldChange } = useForm(config);
+
+  const error = useSelector(({ user: { error } }) => error);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getProfile(data));
```

Comment:
This doesn't seem right. `data` is an empty user object that comes from `useForm`. And if I'm not mistaken you need a user ID. Wha am I missing here?